x-pack/test/security_api_integration/tests/session_concurrent_limit/cleanup.ts

```diff
@@ -158,7 +162,6 @@ export default function ({ getService }: FtrProviderContext) {
 
     beforeEach(async function () {
       this.timeout(120000);
-      await toggleSessionCleanupTask(false);
```

Comment:
__note:__ the main reason why I was disabling task before every test was to make sure the task didn't run in the middle of the test (e.g. we create 3 sessions, we assert that all 3 exist, then we wait until the task removes one of them, and assert that now we have 2 sessions - there was a theoretical risk that the task will run before the first assertion), but if the flaky test runner is OK then it works for us too.

Reply:
The flaky test runner seems to be happy now (yay!) => https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/1786

I could look into this if you like. Otherwise, we can see how likely it will happen. Let me know!

Reply:
> The flaky test runner seems to be happy now (yay!) => https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/1786

Awesome!

> I could look into this if you like. Otherwise, we can see how likely it will happen. Let me know!

Nah, let's see if it ever becomes a problem :+1: